src/main.py

```diff
@@ -29,6 +29,13 @@
     r"^Merge pull request #(?P<pr_number>\d+) from (?:\S)*\n\n(?P<pr_title>.*)$"
 )
 
```

Comment:
The first regex pulls out the first line. To avoid parsing the wrong PR number in edge case scenarios where a different PR number is listed in the description (see commit message for #2506).